frontend/taipy-gui/base/src/app.ts

```diff
@@ -101,7 +101,7 @@ export class TaipyApp {
 
     getFunctionList() {
         const functionData = this.functionData?.getDataTree()[this.context];
-        return functionData ? Object.keys(functionData) : [];
+        return Object.keys(functionData || {});
```

Comment:
Yeah well.. I didn't really like that...

Reply:
I like both ways and Fred is shorter so I chose it :D

Reply:
One could argue that my way is more resource consuming ...